content-src/components/TopSites/TopSites.jsx

```diff
@@ -71,7 +71,9 @@ export class _TopSites extends React.PureComponent {
    * Dispatch session statistics about the quality of TopSites icons and pinned count.
    */
   _dispatchTopSitesStats() {
-    const topSites = this._getVisibleTopSites();
+    const topSites = this._getVisibleTopSites().filter(
+      topSite => topSite !== null && topSite !== undefined
+    );
     const topSitesIconsStats = countTopSitesIconsTypes(topSites);
```

Comment:
SPOC will currently count as a "custom_screenshot", increasing that count by 1. We can add a property here called `spoc or spoc_count`

https://github.com/mozilla/activity-stream/blob/d9ef50f2db4d0be7b7d31a757687586b78e061c1/content-src/components/TopSites/TopSites.jsx#L51-L59

And add a check for link.type === 'SPOC'
https://github.com/mozilla/activity-stream/blob/d9ef50f2db4d0be7b7d31a757687586b78e061c1/content-src/components/TopSites/TopSites.jsx#L21-L24

Reply:
+1 to fix in followup patch to update spoc topsites telemetry